### PR TITLE
Added Admin Login Page

### PR DESCRIPTION
**Pull request by AnaisHebert**
Removed IDIR login from info page
Updated auth routes
Redirect to correct login page on logout

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/auth/auth-routing.module.ts [hunk 1]
```diff
@@ -4,7 +4,10 @@ import { Routes, RouterModule } from '@angular/router';
 import { AuthComponent } from './shared/components/auth/auth.component';
 import { AuthorizationRedirectGuard } from './shared/guards/authorization-redirect.guard';
 
+import { AuthRoutes } from './auth.routes';
+
 import { InfoComponent } from './pages/info/info.component';
+import { AdminComponent } from './pages/admin/admin.component';
 
 const routes: Routes = [
   {
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/auth/auth-routing.module.ts]: The base route for auth can be replaced with the `MODULE_PATH` route constant.

File: prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -62,7 +65,11 @@ export class DashboardComponent implements OnInit {
   }
 
   public onLogout() {
-    this.authService.logout(this.config.loginRedirectUrl);
+    if (this.authService.isAdmin()) {
+      this.authService.logout(this.config.loginRedirectUrl + '/' + AuthRoutes.ADMIN);
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts]: Though there isn't anything wrong with concatenating using + operator in ES6 the backtick can be used to interpolate variables like ``${this.config.loginRedirectUrl}/${AuthRoutes.ADMIN}``

File: prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -84,6 +91,7 @@ export class DashboardComponent implements OnInit {
       .subscribe((device: string) => this.setSideNavProps(device));
 
     const user = await this.authService.getUser();
+    console.log(user);
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts]: Missed a console.log

File: prime-angular-frontend/src/app/modules/auth/auth-routing.module.ts (hunk 1 above)
Comment (AnaisHebert) [reply prime-angular-frontend/src/app/modules/auth/auth-routing.module.ts]: okay!

File: prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts (hunk 2 above)
Comment (AnaisHebert) [reply prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts]: Yes thats better, thanks

File: prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts (hunk 3 above)
Comment (AnaisHebert) [reply prime-angular-frontend/src/app/shared/components/dashboard/dashboard.component.ts]: thanks!

**Pull request merged**